README.md

```diff
@@ -168,3 +168,19 @@ before running `python setup.py install`.
 
 Afterwards, websockify should be available in your path. Run
 `websockify --help` to confirm it's installed correctly.
+
+### Running as a Docker container
+
+By using the [efrecon/websockify](https://hub.docker.com/r/efrecon/websockify/)
```

Comment:
I would prefer to not refer people to outside repositories. In this case I think it would be better to instruct people how they use the `Dockerfile` to get their instance up and running.

Reply:
This is basically what I mentioned in my original message. I think it is important that there is an automated build available and I do not think that it should be hosted under my account, but rather under the novnc umbrella. Creating Docker hub accounts for organisations is well [documented](https://docs.docker.com/docker-hub/orgs/).

Reply:
We should probably look at that as well, but since the README accompanies the source code it should first and foremost be about how to run that source code. Including any local changes.